modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/odbc/OdbcConnectionContext.java

```diff
@@ -153,17 +156,16 @@ public OdbcConnectionContext(GridKernalContext ctx, GridNioSession ses, GridSpin
         String user = null;
         String passwd = null;
 
-        NestedTxMode nestedTxMode = NestedTxMode.DEFAULT;
-
         if (ver.compareTo(VER_2_5_0) >= 0) {
             user = reader.readString();
             passwd = reader.readString();
         }
 
         if (ver.compareTo(VER_2_7_0) >= 0) {
-            byte nestedTxModeVal = reader.readByte();
+            byte nestedTxCode = reader.readByte();
 
-            nestedTxMode = NestedTxMode.fromByte(nestedTxModeVal);
+            if (nestedTxCode != 3)
```

Comment:
Please, omit magic numbers and introduce self explaining constant.
Can be minifed to 

```
if (reader.readByte() != NESTED_TX_ERROR)
    throw new IgniteCheckedException("Nested transactions are not supported!");
```

Reply:
Yep. Fixed the mistake